### DISCUSSION TITLE
Is there any way to enable other TypeScript plugins in `.vue` files?

### DISCUSSION THREAD

**Discussion by aki77** [vuejs/language-tools#483] [Q&A] [answered]
Is there any way to enable other TypeScript plugins in `.vue` files?
The plugins settings listed in `tsconfig.json` are not enabled in `.vue`.

`tsconfig.json`
```json
{
  "compilerOptions": {
    "plugins": [
      {
        "name": "typescript-vue-plugin"
      },
      {
        "name": "typescript-other-plugin"
      }
    ]
  }
}
```

**Comment by johnsoncodehk** [comment, marked as answer]
It depends on what functions other plug-ins provide. Due to the design limitation of TS Plugin, when multiple TS Plugins are set at the same time, the same functions will cover each other instead of cooperating with each other, so it usually does not work as you expect.

**Reply by aki77** [reply]
Thank you for your answer.

For example, the following settings seem to enable the plugin for `.ts` files, but disable it for `.vue` files.
Are you saying that the plugin is enabled in the `.vue` file, but it doesn't work well because the functionality is the same?

```json
    "plugins": [
      {
        "name": "typescript-plugin-namespace-import",
        "options": {
          "paths": [
            "src/logics"
          ]
        }
      }
    ]
```

I have tried the following plugins.
https://github.com/Monchi/typescript-plugin-namespace-import

**Reply by wuliqun** [reply]
I have encountered a similar issue. How can I make the .vue file support the same behaviors as the Tsserver plugin? Can you give me some tips?
